src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -0,0 +1,207 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Create, connect and own instances of data pipeline components.
+
+Provides SDK users direct access to higher level components of the data pipeline,
+eliminating the boiler plate code required to setup the DataSourcingActor and the
+ResamplingActor.
+"""
+
+from __future__ import annotations
+
+import typing
+from dataclasses import dataclass
+
+from frequenz.channels import Broadcast, Sender
+
+# A number of imports had to be done inside functions where they are used, to break
+# import cycles.
+#
+# pylint: disable=import-outside-toplevel
+if typing.TYPE_CHECKING:
+    from ..actor import (
+        ComponentMetricRequest,
+        ComponentMetricsResamplingActor,
+        DataSourcingActor,
+        ResamplerConfig,
+    )
+    from ..timeseries.ev_charger_pool import EVChargerPool
+    from ..timeseries.logical_meter import LogicalMeter
+
+
+@dataclass
+class _ActorInfo:
```

Comment:
you might want to add a brief line of documentation for this class

Reply:
done.